### PR TITLE
Project msbuild syntax cleanup

### PR DESCRIPTION
**Pull request by ViktorHofer**
Those attributes aren't needed anymore with the portable msbuild + our min version of msbuild desktop 15. Also removed the license headers that I added but which aren't needed as the marked files aren't shipping artifacts.

cc @ericstj @safern

### CODE REVIEW

**Comment by danmosemsft** [comment]
Good to see you're using the new function https://github.com/Microsoft/msbuild/issues/1277

**Comment by ViktorHofer** [comment]
Unfortunately that function has a bug which makes it not very useful: https://github.com/Microsoft/msbuild/issues/2456#issuecomment-468642451

**Comment by ViktorHofer** [comment]
Yes I did regex replaces and checked most of it manually

File: eng/Packaging.props [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="Build">
```
Comment (ericstj) [inline eng/Packaging.props]: You can remove DefaultTargets="Build", especially from props/targets files.

File: Directory.Build.targets [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
```
Comment (safern) [inline Directory.Build.targets]: I think we can also delete the <?xml ?> entries, right @ericstj ?

Comment (ericstj) [reply Directory.Build.targets]: Good point.  MSBuild doesn't require this anymore.  I guess it depends on wether or not all our tools which try to edit these are happy without the prolog.  Seems like we'd be ok removing it and see what breaks and add it back.  The only things that might break are maestro-like tools I think, or any crusty dev tools that are accessing projects without using MSBuild.

File: eng/Packaging.props (hunk 1 above)
Comment (ViktorHofer) [reply eng/Packaging.props]: I removed that attribute from all props and targets file.

File: Directory.Build.targets (hunk 2 above)
Comment (ViktorHofer) [reply Directory.Build.targets]: Good to know, I wasn't sure if we still need them. Let's see.

File: src/Common/tests/Resources/Strings.resx [hunk 3]
```diff
@@ -1,5 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (safern) [inline src/Common/tests/Resources/Strings.resx]: I'm not sure if for .resx files it was still needed. @ericstj do you know?

Comment (ericstj) [reply src/Common/tests/Resources/Strings.resx]: Not off hand.  Test VS resx designer (for editing) and build (for reading in our code generator and compilation in resgen).

**Pull request merged**